pkg/reconciler/events/pubsub/pubsub.go

```diff
@@ -111,3 +130,54 @@ func (r *Reconciler) reconcilePullSubscription(ctx context.Context, source *v1al
 	}
 	return ps, nil
 }
+
+func (r *Reconciler) CreateServiceAccount(ctx context.Context, pubsub *v1alpha1.CloudPubSubSource) (*corev1.ServiceAccount, pkgreconciler.Event) {
```

Comment:
I would still return an error instead of Event... go does this magic wrapping thing, but I think is clearer if in the signature we put error. I'd rather change the signature everywhere but in ReconcileKind

Reply:
Yeah, I'll follow this to return error instead of Event.